src/lib/chain/commitment.ts

```diff
@@ -440,13 +452,88 @@ export function ccStatusToString(status: bigint | undefined) {
   return (
     (
       [
+        "Inactive",
         "Active",
         "WaitDelegation",
         "WaitStart",
-        "Inactive",
         "Failed",
         "Removed",
```

Comment:
I'm surprised that you need to define this constant by yourself, instead of taking it from ts-client.

Reply:
https://fluencelabs.slack.com/archives/C023BPPBM25/p1711008075941139